css/list.css

```diff
@@ -0,0 +1,64 @@
+.list{
+    padding: 0;
+    padding-left: 1rem;
+}
+
+.list > li{
+    line-height: 1.5;
+}
+
+.ul-disc{
+    list-style-type: disc;
+}
+
+.ul-square{
+    list-style-type: square;
+}
+
+.ul-circle{
+    list-style-type: circle;
+}
+
+.ol-number{
+    list-style-type: decimal;
+}
+
+.ol-lower-alpha{
+    list-style-type: lower-alpha;
+}
+
+.ol-upper-alpha{
+    list-style-type: upper-alpha;
+}
+
+.ol-lower-roman{
+    list-style-type: lower-roman;
+}
+
+.ol-upper-roman{
+    list-style-type: upper-roman;
+}
+
+.no-bullets{
+    list-style-type: none;
+    padding: 0;
+}
+
+.list.inline{
+    display: flex;
+    list-style-type: none;
+    padding: 0;
+    gap: 1rem;
+}
+
+.list.stacked{
+    list-style-type: none;
+    padding: 0;
+}
```

Comment:
/* list-style-type: none;
padding: 0; */

can be avoided (code same as .no-bullets [line-42] )